src/MixedContentExtractor.php

```diff
@@ -15,6 +15,15 @@ public static function extract(string $html, string $currentUri): array
             ->mapSpread(function ($tagName, $attribute) use ($html, $currentUri) {
                 return (new DomCrawler($html, $currentUri))
                     ->filterXPath("//{$tagName}[@{$attribute}]")
+                    ->reduce(function (DomCrawler $node) {
+                        $relAttr = $node->getNode(0)->attributes->getNamedItem('rel');
+
+                        if ($relAttr !== null && strtolower($relAttr->nodeValue) === 'shortlink') {
+                            return false;
+                        }
+
+                        return true;
```

Comment:
This line isn't 100% needed. My PhpStorm complained about 'you must return something here'. So feel free to remove it, it will work regardless.